cmd/crc/cmd/start.go

```diff
@@ -182,8 +191,12 @@ func getPullSecretFileContent() (string, error) {
 		err        error
 	)
 
-	// In case user doesn't provide a file in start command or in config then ask for it.
-	if config.Get(cmdConfig.PullSecretFile).AsString() == "" {
+	if crcversion.IsOkdBuild() {
+		// If crc is built from an OKD bundle, then use the fake pull secret in contants.
+		pullsecret = constants.OkdPullSecret
+
+		// In case user doesn't provide a file in start command or in config then ask for it.
+	} else if config.Get(cmdConfig.PullSecretFile).AsString() == "" {
```

Comment:
Beware `config.Get(cmdConfig.PullSecretFile)` is used in 3 places.

Reply:
The other call outside of the if/else block is in init().  I don't believe that it interferes with the logic for an OKD bundle build.